test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -5557,8 +5557,8 @@ public async Task CustomSerializerSettings_TypeNameHandlingAll()
                 var status = await client.WaitForCompletionAsync(this.output);
 
                 Assert.Equal(OrchestrationRuntimeStatus.Completed, status?.RuntimeStatus);
-                Assert.Equal("World", status?.Input);
-                Assert.Equal("Hello, World!", status?.Output);
+                Assert.Equal("World", (string)status?.Input);
+                Assert.Equal("Hello, World!", (string)status?.Output);

```

Comment:
I ran into an issue in an earlier iteration that caused this test to fail. Unfortunately, xunit doesn't know how to display `JValue` objects in the error messages (it always just shows "[]") so the test output was unfortunately useless. Casting the `JValue` properties to `string`, however, fixes the display problem.